templates/layout/_options_appearance.html.twig

```diff
@@ -17,16 +17,16 @@
         {{ component('settings_row_switch', {label: 'show_top_bar'|trans, settingsKey: 'KBIN_GENERAL_TOPBAR'}) }}
 
         {% if app.user is defined and app.user is not same as null %}
-            {{ component('settings_row_switch', {label: 'show_subscriptions'|trans, settingsKey: 'KBIN_GENERAL_SHOW_SUBSCRIPTIONS'}) }}
-            {% if app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_SHOW_SUBSCRIPTIONS')) is same as 'true' %}
+            {{ component('settings_row_switch', {label: 'show_subscriptions'|trans, settingsKey: 'KBIN_GENERAL_SHOW_SUBSCRIPTIONS', defaultValue: 'true'}) }}
+            {% if app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_SHOW_SUBSCRIPTIONS')) is not same as 'false' %}
                 {{ component('settings_row_enum', {label: 'subscription_sort'|trans, settingsKey: 'KBIN_GENERAL_SHOW_SUBSCRIPTIONS_SORT', values: [ {name: 'alphabetically'|trans , value: 'ALPHABETICALLY'}, {name: 'last_active'|trans , value: 'LAST_ACTIVE' } ], defaultValue: 'LAST_ACTIVE'}) }}
-            {% endif %}
-            {% if app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_SHOW_SUBSCRIPTIONS')) is same as 'true' %}
                 {{ component('settings_row_switch', {label: 'subscriptions_in_own_sidebar'|trans, settingsKey: 'KBIN_GENERAL_SHOW_SUBSCRIPTIONS_IN_SEPARATE'}) }}
```

Comment:
I thought it would be to make the settings appear to more users. But I think you're right and they could be removed.

Reply:
I think this is my only outstanding comment: I would say get rid of those two options in the sidebar options as they've moved to the widget itself

Reply:
I'll do that then

Reply:
wait which one? :stuck_out_tongue_closed_eyes: if the latter and hiding it on mobile, I did that with the page width toggle in this same file, you could potentially rename that from `width-setting` to something generic like `hide-on-mobile` and use it for all 3 toggles. if the former can ignore this comment

Reply:
I just hid the two options, but now I realize, that you have no way of making the sidebar appear at the top on mobile, because you can't make the sidebar separate from the rest :D

Reply:
If I could easily just always put the sidebar on top if the user is on mobile that would be very nice... But nobody told me an easy way to recognize mobile user agents :sweat_smile: